### DISCUSSION TITLE
Dynamic/interpolated custom confirmation prompt

### DISCUSSION THREAD

**Discussion by mway** [casey/just#1854] [Ideas]
AFAICT this isn't currently possible, but it would be nice if custom confirmation prompts supported dynamic values based on e.g. recipe parameters (or other available variables).

For example:

```just
[confirm("Are you sure you want to foo {{ quote(BAR) }}?")]
foo BAR:
    # ...
```

or

```just
[confirm({{ /* something that produces a string */ }})]
foo BAR:
    # ...
```

This is mostly based on the idea that if something is important enough to confirm, it might be helpful to allow recipe authors to provide more context within the confirmation dialog (versus a universal "Are you sure?").

**Comment by casey** [comment]
Seems reasonable enough! I opened #1855 to track this.
